ovos_number_parser/util.py

```
@@ -211,3 +258,597 @@ def convert_to_mixed_fraction(number, denominators=range(1, 21)):
+        # If there's no remainder, we're done
+        if remainder == 0:
+            return count_str
+
+        # Pronounce the remainder and join
+        remainder_str = self.pronounce_ordinal(remainder, gender, scale)
+
+        return f"{count_str} {remainder_str}"
+
+    def _pronounce_up_to_999(self,
+                             n: int,
+                             gender: GrammaticalGender = GrammaticalGender.MASCULINE
```

Ruff (0.14.7)

Unused method argument: `gender`

(ARG002)